l10n_it_partially_deductible_vat/AUTHORS.txt

```diff
@@ -0,0 +1,2 @@
+Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+Franco Tampieri <franco.tampieri@agilebg.com>
```

Comment:
Please remove this file and use README template available at
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst